### PR TITLE
[telemetry-service] add feature flag for location labels

### PR DESCRIPTION
**Pull request by aluon**
## Description

Put the location labels behind a feature flag so we can we can enable/disable this without changing code

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## How Has This Been Tested?

CI

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: crates/aptos-telemetry-service/src/prometheus_push_metrics.rs [hunk 1]
```diff
@@ -45,6 +45,10 @@ pub async fn handle_metrics_ingest(
 ) -> anyhow::Result<impl Reply, Rejection> {
     debug!("handling prometheus metrics ingest");
 
+    let enable_location_labels = env::var("FEATURE_LOCATION_LABELS_ENABLED")
+        .map(|val| val.parse::<bool>().unwrap_or(false))
+        .unwrap_or(false);
+
     let enable_random_label = env::var("FEATURE_RANDOM_LABEL_ENABLED")
```
Comment (ibalajiarun) [inline crates/aptos-telemetry-service/src/prometheus_push_metrics.rs:52]: Did we decide to keep this feature?

**Review by ibalajiarun — APPROVED** [review]

File: crates/aptos-telemetry-service/src/prometheus_push_metrics.rs (hunk 1 above)
Comment (aluon) [reply crates/aptos-telemetry-service/src/prometheus_push_metrics.rs:52]: I'm planning to keep this disabled by default. I still haven't figured out the issue where the labels alternate between location and no location

Comment (ibalajiarun) [reply crates/aptos-telemetry-service/src/prometheus_push_metrics.rs:52]: I meant the `FEATURE_RANDOM_LABEL_ENABLED`?

Comment (aluon) [reply crates/aptos-telemetry-service/src/prometheus_push_metrics.rs:52]: Oh, I think this still might be useful to keep around for loadtesting our selfhosted VM instance

**Review by sherry-x — APPROVED** [review]

**Pull request merged**